openfl/component/director/director.py

```diff
@@ -202,6 +206,71 @@ def run_tensorboard(self):
             self.tensorboard_thread.start()
         except Exception as exc:
             logger.error(f'Failed to run tensorboard: {exc}')
+    
+    def check_kill_process(self, pstring):
+        """Kill process by name."""
+        import signal
+
+        for line in os.popen("ps ax | grep " + pstring + " | grep -v grep"):
+            fields = line.split()
+            pid = fields[0]
+            os.kill(int(pid), signal.SIGKILL)
+
+    def create_dirs(self):
+        """Create CA directories."""
+        prefix = Path('.')
+        (prefix / self.pki_dir).mkdir(parents=True, exist_ok=True)
+        (prefix / self.step_config_dir).mkdir(parents=True, exist_ok=True)
+
+    def run_ca(self, password, ca_url):
```

Comment:
I believe this code is external to the director component. Probably will move this to the ca component as well.
Moreover, looks like it costs nothing to start ca on a separate machine.